media-video/mplayer/mplayer-1.4.ebuild

```diff
@@ -0,0 +1,634 @@
+	if [[ ${PV} = *9999* ]]; then
+		# Set SVN version manually
+		subversion_wc_info
+		printf "${ESVN_WC_REVISION}" > $svf
+	fi
+	if [[ ! -f VERSION ]] ; then
+		[[ -f "$svf" ]] || die "Missing ${svf}. Did you generate your snapshot with prepare_mplayer.sh?"
+		local sv=$(<$svf)
+		printf "SVN-r${sv} (Gentoo)" > VERSION
+	fi
+
+	# fix path to bash executable in configure scripts
+	sed -i -e "1c\#!${EPREFIX}/bin/bash" configure version.sh || die
+
+	# Use sane default for >=virtual/udev-197
+	sed -i -e '/default_dvd_device/s:/dev/dvd:/dev/cdrom:' configure || die
+}
+
+src_configure() {
+	local myconf=""
+	local uses i
+
+	# set LINGUAS
+	[[ -n $LINGUAS ]] && LINGUAS="${LINGUAS/da/dk}"
+	[[ -n $LINGUAS ]] && LINGUAS="${LINGUAS/zh/zh_CN}" #482968
+
+	# mplayer ebuild uses "use foo || --disable-foo" to forcibly disable
+	# compilation in almost every situation. The reason for this is
+	# because if --enable is used, it will force the build of that option,
+	# regardless of whether the dependency is available or not.
+
+	###################
+	#Optional features#
+	###################
+	# disable svga since we don't want it
+	# disable arts since we don't have kde3
+	# always disable internal ass
+	# disable opus and ilbc since it only controls support in internal
+	#         ffmpeg which we do not use
+	myconf+="
+		--disable-svga --disable-svgalib_helper
+		--disable-ass-internal
+		--disable-arts
+		--disable-directfb
+		--disable-kai
+		--disable-libopus
+		--disable-libilbc
+		--disable-xvmc
+		$(use_enable network networking)
+		$(use_enable joystick)
+	"
+	uses="bl bluray enca ftp rtc vcd" # nemesi <- not working with in-tree ebuild
+	myconf+=" --disable-nemesi" # nemesi automagic disable
+	for i in ${uses}; do
+		myconf+=" $(usex ${i} '' --disable-${i})"
+	done
+	myconf+=" $(usex bidi '' --disable-fribidi)"
+	myconf+=" $(usex ipv6 '' --disable-inet6)"
+	myconf+=" $(usex libass '' --disable-ass)"
+	myconf+=" $(usex nut '' --disable-libnut)"
+	myconf+=" $(usex rar '' --disable-unrarexec)"
+	myconf+=" $(usex samba '' --disable-smb)"
+	myconf+=" $(usex lirc '' '--disable-lirc --disable-lircc --disable-apple-ir')"
+
+	# enable openssl support for https
+	# the binary cannot be redistributed if set
+	# by default gnutls will be enabled
+	if use ssl ; then
+		myconf+=" $(usex !gnutls '--disable-gnutls --enable-openssl-nondistributable' '')"
+	else
+		myconf+=" --disable-gnutls"
+	fi
+
+	# libcdio support: prefer libcdio over cdparanoia
+	# don't check for cddb w/cdio
+	if use cdio; then
+		myconf+=" --disable-cdparanoia"
+	else
+		myconf+=" --disable-libcdio"
+		myconf+=" $(usex cdparanoia '' --disable-cdparanoia)"
+		myconf+=" $(usex cddb '' --disable-cddb)"
+	fi
+
+	################################
+	# DVD read, navigation support #
+	################################
+	#
+	# dvdread - accessing a DVD
+	# dvdnav - navigation of menus
+	myconf+= " $(usex dvd '' --disable-dvdread)"
+	myconf+= " $(usex dvdnav '' --disabledvdnav)"
```

Comment:
Most likely the extra space after the `myconf+=` in these two lines.

Reply:
wouldn't it likely be `--disable-dvdnav` instead?